pkg/bench/rttanalysis/testdata/benchmark_expectations

```diff
@@ -77,6 +77,7 @@ exp,benchmark
 12,ORMQueries/hasura_column_descriptions_8_tables
 5,ORMQueries/hasura_column_descriptions_modified
 4,ORMQueries/information_schema._pg_index_position
+0,ORMQueries/column_descriptions_json_agg
```

Comment:
this is interesting, 0 round trips...is it because all the pg tables are looking at cached descriptors?

Reply:
i actually am not entirely sure.. but i guess so. in this case, i added the benchmark primarily so that latency regressions would be revealed. (test-eng has plans to automatically monitor latency changes in microbenchmarks)